security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -521,6 +526,22 @@ func (sc *SecretCache) generateSecret(ctx context.Context, token, resourceName s
 		certChain = append(certChain, []byte(c)...)
 	}
 
+	// Cert exipre time by default is createTime + sc.configOptions.SecretTTL.
```

Comment:
Add more comment to explain "by default" behavior and non-default behavior?

Reply:
done